src/Microsoft.Graph/Generated/model/ConnectorGroup.cs

```diff
@@ -64,7 +64,7 @@ public ConnectorGroup()
         /// Gets or sets members.
         /// </summary>
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "members", Required = Newtonsoft.Json.Required.Default)]
-        public IConnectorGroupMembersCollectionWithReferencesPage Members { get; set; }
+        public IConnectorGroupMembersCollectionPage Members { get; set; }
```

Comment:
What happens when ContainsTarget = true is added to a navigation to a collection